libmscore/measure.cpp

```diff
@@ -2588,10 +2596,10 @@ void Measure::scanElements(void* data, void (*func)(void*, Element*), bool all)
 
       int nstaves = score()->nstaves();
       for (int staffIdx = 0; staffIdx < nstaves; ++staffIdx) {
-            if (!all && !(visible(staffIdx) && score()->staff(staffIdx)->show()))
+            if (!all && !score()->staff(staffIdx)->show())
```

Comment:
What happens if show is called while staffIdx is not visible?

Reply:
No problem, it's just a check if the measure is set as invisible. Not related to the exception I'm chasing. It was introduced before this PR.